PowerShell/JumpCloud Module/Public/Administrators/Get-JCAdmin.ps1

```diff
@@ -0,0 +1,97 @@
+function Get-JCAdmin {
+    param (
+        [Parameter(ValueFromPipelineByPropertyName, Position = 0, HelpMessage = 'The email of the JumpCloud admin you wish to search for.')]
+        [String]$email,
+        [Parameter(ValueFromPipelineByPropertyName, Position = 1, HelpMessage = 'A search filter to search for admins with multifactor enabled/disabled.')]
+        [Boolean]$enableMultifactor,
+        [Parameter(ValueFromPipelineByPropertyName, Position = 2, HelpMessage = 'A search filter to search for admins with totp enabled/disabled.')]
+        [Boolean]$totpEnrolled,
+        [Parameter(ValueFromPipelineByPropertyName, Position = 3, HelpMessage = 'A search filter to search for admins based on their role')]
+        [String]$roleName,
+        [Parameter(ValueFromPipelineByPropertyName, Position = 4, HelpMessage = 'A search filter to search for admins based on their organization (Only for MTP/MSP tenants)')]
+        [String]$organization
```

Comment:
My only suggestion here would be to change the paramName to be "organizationID" because technically we are asking for an ID value, I feel like we should call that out.